eq-author/src/App/section/Design/index.js

```diff
@@ -178,6 +178,7 @@ export const UnwrappedSectionRoute = (props) => {
       preview={hasIntroductionContent}
       title={section.displayName || ""}
       validationErrorInfo={section.validationErrorInfo}
+      logic
```

Comment:
```suggestion
      logic = {enableOn(['hub'])}
```

Otherwise, we'll be able to see the logic page on V2 as well.

You'll need to import the enableOn function from the feature flags, too.